Issue 15: mention checksums in hybrid simulations

teto opened the pull request:
In case of hybrid simulations Kernel drops packets (e.g. sends RST) if ns3 checksums are disabled.
Add a mention in documentation.

I am not sure it's the best place to put the mention ?

== EDIT
Hybrid simulations refer to simulations involving both ns and linux or BSD stacks. I would like to raise in the doc points of importance to get reasonable results.
- [x] ns Checksums need to be enabled
- [ ] ns3 callbacks don't get called https://github.com/direct-code-execution/ns-3-dce/issues/24
- [ ] List ns attributes that need to change (Dunno if an ns3 page exist, for instance ns3 and linux TCP RTOs differ).

thehajime commented:
thanks for the PR.

> I am not sure it's the best place to put the mention ?

I think it can be documented in the following file, either inside the limitation section or new one.

https://github.com/direct-code-execution/ns-3-dce/blob/master/doc/source/dce-user-tech.rst

teto committed:
Issue 15: mention checksums in hybrid simulations

In case of hybrid simulations Kernel drops packets (e.g. sends RST) if ns3 checksums are disabled.
Add a mention in documentation.

teto commented:
In fact I believe it makes sense to have an hybrid section because there are several tips/tricks onbe should be aware of such as : 
https://github.com/direct-code-execution/ns-3-dce/issues/24
and also the routing interface is not unified which is a bit annoying.

thehajime commented:
sorry, I don't understand what a **hybrid section** is. care to elaborate ?

teto commented:
I am thinking of scenarios involving both ns3 and linux or BSD stacks. If
you can come up with a better name :)

2016-02-03 0:56 GMT+01:00 Hajime Tazaki <notifications@github.com>:

> sorry, I don't understand what a *hybrid section* is. care to elaborate ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/direct-code-execution/ns-3-dce/pull/31#issuecomment-178893866>
> .
>

thehajime commented:
okay.

I think for the moment we can fix one-by-one: once we come up with the nice text then we can create a new section and put this patch to the section. is it fine for you ?

teto commented:
ok I'll add items to the section (dunno when though) and you deem it ready, we can merge.
